www/src/components/horizontal-nav-list.js

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import Link from "gatsby-link"
+import styled from "@emotion/styled"
+
+import { colors, mediaQueries } from "../utils/presets"
+
+const LinkList = styled.ul`
+  margin: 0;
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  * + * {
+    border-left: 1px solid ${colors.ui.border.subtle};
+  }
+`
+const Li = styled.li`
+  padding: 3px 12px;
+  ${mediaQueries.md} {
+    padding: 0px 6px;
+  }
+  margin: 0;
+  &:hover {
+    background-color: ${colors.ui.hover};
+  }
+`
+
+const HorizontalNavList = ({ items = [], slug }) => (
+  <nav>
+    <LinkList>
+      {items.map(item => (
+        <Link key={item} to={`${slug.slice(0, -1)}#${item.toLowerCase()}`}>
```

Comment:
The Link needs to go inside of the Li to properly structure a UL. So it would look like this:
```
<LinkList>
    {items.map(item => (
        <Li key={item}>
            <Link to={`${slug.slice(0, -1)}#${item.toLowerCase()}`}>{item}</Link>
        </Li>
    ))}
</LinkList>
```

Reply:
TIL! I guess I just always assumed a `<li>` could be any number of levels of descendancy deep in a list. Thanks for pointing this out rather than just correcting it yourself 🙂 

Fixing now...

Reply:
You can put lists inside of list items, but they have to be properly structured with parents > children. You can read more here, note the "permitted content" section: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul